### ISSUE TITLE
About training results

### ISSUE THREAD

**Issue by alan525** [neuralchen/SimSwap#72]
I trained the model on my own data and adopted the loss function consistent with the paper. The loss function was stable soon, but the generated pictures were not good. I wonder whether this result is normal and whether it needs to be trained for a long time.
![image2](https://user-images.githubusercontent.com/17896521/127125495-b693d2fe-9b17-4478-9739-7f0f346f07ec.png)
